network-wrapper

```diff
@@ -153,15 +157,15 @@ done < /proc/mounts
 }
 
 # Try to find a cgroup matching exactly the provided name.
-N=$(find "$CGROUPMNT" -name "$GUESTNAME" | wc -l)
+N=$(find "$CGROUPMNT" -name "$GUEST_LABEL_NAME" | wc -l)
```

Comment:
This needs to be tested in both centos and ubuntu, the cgroup will have container id and not name.

Reply:
Have not verified in Centos.